src/itype.h

```diff
@@ -1411,12 +1411,5 @@ struct itype {
 
 void load_charge_removal_blacklist( const JsonObject &jo, const std::string &src );
 void load_charge_migration_blacklist( const JsonObject &jo, const std::string &src );
-// can be removed once all known bad items got fixed
-class known_bad_density
```

Comment:
I'll note that we are losing some context from comments in a few places.

Reply:
Yeah, I judged the comment isn't really relevant enough when it's not it's own class anymore. I can readd it if people think it should still be there.

Reply:
Added a comment.